.circleci/config.yml

```diff
@@ -0,0 +1,53 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+    build:
+        branches:
+            ignore:
+                - gh-pages
+        docker:
+            # specify the version you desire here
+            - image: circleci/node:9-browsers
+
+        working_directory: ~/repo
+
+        steps:
+            - checkout
+
+            # Download and cache dependencies
+            - restore_cache:
+                keys:
+                    - v1-dependencies-{{ checksum "package.json" }}
+                    # fallback to using the latest cache if no exact match is found
+                    - v1-dependencies-
+            - run:
+                name: Install dependencies
+                command: npm install
+
+            - save_cache:
+                paths:
+                    - node_modules
+                key: v1-dependencies-{{ checksum "package.json" }}
+
+            # run tests
+            - run:
+                name: Run tests
+                command: npm test --includecoretests
+                # command: npm run test
+
+            # run build
+            - run:
+                name: Run build
+                command: npm run build
+
+            - add_ssh_keys:
+                fingerprints:
+                    - "2a:ed:6a:f3:34:9d:7a:17:e3:75:74:2d:4f:ab:75:7d"
```

Comment:
these look so secretive, but I'm assuming this is more or less a public key?

Reply:
https://en.wikipedia.org/wiki/Public_key_fingerprint

Reply:
right, just making sure